packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -30,6 +30,17 @@ const stubLocalCameraName = async (page: Page): Promise<void> => {
   });
 };
 
+/* @conditional-compile-remove-from(stable) meeting/calling-composite <Local-Camera-Switcher> */
```

Comment:
You shouldn't have to conditionally compile this. The check itself should return true/false unconditionally.

Reply:
so should I just conditionally compile the individual test?

Reply:
You can conditionally compile the test suite. I just meant that this helper function doesn't need to be conditional compiled.

Is the problem that you aren't exporting the function so lint complains that it is unused? It should anyway be moved to (and exported from) a test helper file in `common/utils`